Devscord.DiscordFramework/Framework/Commands/Services/BotCommandsMatchingService.cs

```diff
@@ -31,7 +31,7 @@ public bool IsMatchedWithNormalCommand(BotCommandTemplate template, IEnumerable<
             {
                 throw new NotEnoughArgumentsException();
             }
-            var argsAndValues = arguments.Select(arg => new KeyValuePair<string, string>(arg.Name, arg.Value));
+            var argsAndValues = arguments.Select(arg => new KeyValuePair<string, string>(arg.Name, arg.Value)).ToList();
```

Comment:
jeśli dobrze patrzę to tego argsAndValues używasz tylko raz poźniej, więc ToList nie jest potrzebne

Reply:
ale jest w pętli przetwarzane to więc nie wiem czy to nie będzie zrobione w taki sposób, że zostanie to obliczone tyle razy ile pętla się wykona

Reply:
masz racje, zostaw tak jak jest